src/lib.rs

```diff
@@ -434,61 +501,61 @@ make_color! {
     ///Linear luminance.
     Luma {
         ///Linear luminance.
-        y(luma: T)[alpha: T] => new;
+        y(luma: T)[alpha: T] => with_wp;
```

Comment:
The constructors are currently implemented as:

```rust
impl<Wp, T> Color<Wp, T>
    where T: Float,
        Wp: WhitePoint<T>
{
    $(
        $(
            #[$ctor_comment]
            pub fn $ctor_name$(<$($ty_params : $ty_param_traits$( <$( $ty_inner_traits ),*> )*),*>)*($($ctor_field: $ctor_ty),*) -> Color<Wp, T> {
                Color::$variant($variant::$ctor_original($($ctor_field),*))
            }
        )+
    )+
}

///<span id="Colora"></span>[`Colora`](type.Colora.html) implementations.
impl<Wp, T> Alpha<Color<Wp, T>, T>
    where T: Float,
        Wp: WhitePoint<T>
{
    $(
        $(
            #[$ctor_comment]
            pub fn $ctor_name$(<$($ty_params : $ty_param_traits$( <$( $ty_inner_traits ),*> )*),*>)*($($ctor_field: $ctor_ty,)* alpha: $alpha_ty) -> Colora<Wp, T> {
                Alpha::<$variant<Wp, T>, T>::$ctor_original($($ctor_field,)* alpha).into()
            }
        )+
    )+
}
````

I see no reason why it can't be changed to this:

```rust
impl<T: Float> Color<D65, T> {
    $(
        $(
            #[$ctor_comment]
            pub fn $ctor_name$(<$($ty_params : $ty_param_traits$( <$( $ty_inner_traits ),*> )*),*>)*($($ctor_field: $ctor_ty),*) -> Color<D65, T> {
                Color::$variant($variant::$ctor_original($($ctor_field),*))
            }
        )+
    )+
}

///<span id="Colora"></span>[`Colora`](type.Colora.html) implementations.
impl<T: Float> Alpha<Color<D65, T>, T> {
    $(
        $(
            #[$ctor_comment]
            pub fn $ctor_name$(<$($ty_params : $ty_param_traits$( <$( $ty_inner_traits ),*> )*),*>)*($($ctor_field: $ctor_ty,)* alpha: $alpha_ty) -> Colora<D65, T> {
                Alpha::<$variant<D65, T>, T>::$ctor_original($($ctor_field,)* alpha).into()
            }
        )+
    )+
}
````